terraform-incubator/access-the-data/main.tf

```diff
@@ -0,0 +1,148 @@
+locals {
```

Comment:
The contents of this file were designed to contain exactly what varies between projects. Other projects might crib from this, but expect to change basically every value.